web/modules/custom/nlc_prototype/js/nlc_prototype.google_analytics.js

```diff
@@ -42,6 +42,14 @@
           }
         );
       }
+
+      if (typeof settings.path.currentPathIsSelfProfile !== 'undefined' && settings.path.currentPathIsSelfProfile === true) {
+        gtag('event', 'PageView', {
+          event_category: 'Own profile page',
```

Comment:
I think it would make sense to add `'non_interaction': true` for this event, as they aren't doing anything apart from viewing this page, we don't want it to count as a second interaction.